src/clarity/clarityValue.ts

```diff
@@ -45,6 +47,40 @@ export type ClarityValue =
   | ListCV
   | TupleCV;
 
+export function cvToString(val: ClarityValue, encoding?: 'ascii' | 'hex'): string {
+  switch (val.type) {
+    case ClarityType.BoolTrue:
+      return 'true';
+    case ClarityType.BoolFalse:
+      return 'false';
+    case ClarityType.Int:
+      return val.value.fromTwos(CLARITY_INT_SIZE).toString();
+    case ClarityType.UInt:
+      return val.value.toString();
+    case ClarityType.Buffer:
+      const bufferString =
+        encoding === 'hex' ? val.buffer.toString('hex') : val.buffer.toString('ascii');
```

Comment:
Can you add an `encoding` option that is something like `tryAscii`? Until we have a proper string type implemented in Clarity, we settled on making the front-end display buffers as ascii strings _if_ they can be decoded into a valid Clarity string literal

Reply:
Also I think for hex it needs to be `0x${bufferString}` (unquoted)